components/reddit/sources/new-comments-by-user/new-comments-by-user.js

```diff
@@ -28,25 +28,22 @@ module.exports = {
   hooks: {
     async deploy() {
       // Emits sample events on the first run during deploy.
-      let before = null;
       var redditComments = await this.reddit.getNewUserComments(
-        before,
+        null,
         this.username,
         this.numberOfParents,
         this.timeFilter,
         this.includeSubredditDetails,
         10
       );
-      if (!redditComments) {
-        console.log("No data available, skipping emitting sample events");
+      const { children: comments = [] } = redditComments.data;
+      if (comments.length === 0) {
+        console.log("No data available, skipping itieration");
```

Comment:
Noticed there's a typo here (it's repeated in a few places). Itieration should be iteration.

Reply:
They've been corrected.